src/Symfony/Component/HttpClient/Chunk/ServerSentEvent.php

```diff
@@ -76,4 +78,30 @@ public function getRetry(): float
     {
         return $this->retry;
     }
+
+    /**
+     * Gets the SSE data decoded as an array when it's a JSON payload.
+     */
+    public function getArrayData(): array
```

Comment:
I prefer `getArrayData()` because the SSE carries more info than its data, so `toArray()` could return `['id' => 'foo', 'type' => 'bar', ...]` and it stays consistent with `getData()`.

Reply:
Makes sense